doc/changelog/10-standard-library/11686-fix-int-notations.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  Notations [|x|] and [||x||] for morphisms from 63-bit integers to Z and
```

Comment:
At the very least you need some quoting here like so:

```rst
   Notations ``[|x|]`` and ``[||x||]`` for morphisms from 63-bit integers to Z and
```

but the best would be to use a notation like so:

```rst
   Notations :n:`[|@ident|]` and :n:`[||@ident||]` for morphisms from 63-bit integers to :g:`Z` and
   :g:`zn2z int` have been removed in favor of :n:`φ @ident` and :n:`Φ @ident` respectively. These

Reply:
Thanks, I'll prepare a fix. We should also investigate why there were so many spurious failures on the CI for this PR, it made @thery miss the real problem. Even the doc build was not showing the real failure.